static/lang/re-en.json

```diff
@@ -3884,7 +3884,24 @@
+                "QuickenedCastingWitch": "You accelerate your spellcasting. If your next action is to cast a witch cantrip or a witch spell that is at least 2 ranks lower than the highest-rank witch spell slot you have, reduce the number of actions to cast it by 1 (minimum 1 action).",
+                "QuickenedCastingWizard": "Straining your mind, you collapse as much of your spell's formulas as you can, resulting in a shorter but much more complex incantation. If your next action is to cast a wizard cantrip or a wizard spell that is at least 2 ranks lower than the highest-rank wizard spell slot you have, reduce the number of actions to cast it by 1 (minimum 1 action)",
+                "ReachSpell": "You can extend the range of your spells. If the next action you use is to Cast a Spell that has a range, increase that spell's range by 30 feet. As is standard for increasing spell ranges, if the spell normally has a range of touch, you extend its range to 30 feet.",
+                "ScintillatingSpell": "Your spells become a radiant display of light and color. If your next action is to Cast a Spell that doesn't have the darkness trait, has no duration, and requires creatures to attempt a Reflex save, the spell explodes in a spray of scintillating lights, in addition to its other effects. Each creature that failed its Reflex save against the spell is @UUID[Compendium.pf2e.conditionitems.Item.TkIyaNPgTZFBCCuh]{Dazzled} for 1 round, and those who critically failed are instead @UUID[Compendium.pf2e.conditionitems.Item.XgEqL1kFApUbl5Z2]{Blinded} for 1 round.",
+                "SecondaryDetonationArray": "You divert some of your spell's energy into an unstable runic array. If your next action is to Cast a Spell that deals damage, has no duration, and affects an area, a glowing magic circle appears in a @Template[type:burst|distance:5] within that area. At the beginning of your next turn, the circle detonates, dealing @Damage[(@item.level)d6[force]] damage per rank of the spell to all creatures within the circle, with a basic Reflex save against your spell DC. If the spell dealt a different type of damage, the circle deals this type of damage instead (or one type of your choice if the spell could deal multiple types of damage @Damage[(@item.level)d6[bludgeoning]], @Damage[(@item.level)d6[piercing]], @Damage[(@item.level)d6[slashing]], @Damage[(@item.level)d6[spirit]], @Damage[(@item.level)d6[poison]],  @Damage[(@item.level)d6[acid]], @Damage[(@item.level)d6[cold]], @Damage[(@item.level)d6[electricity]], @Damage[(@item.level)d6[fire]], @Damage[(@item.level)d6[sonic]], @Damage[(@item.level)d6[vitality]], @Damage[(@item.level)d6[void]])"
```

Comment:
Not sold appending all these buttons here. We should be able to resolve the spell's damage from its data in a damage button, or if nothing else, use a second suboption to pick a damage type.

Reply:
Fair.

Think I'll go with the second suboption as the feat allows you to pick from multiple damages if the spell does so.

Reply:
Yeah, unfortunately no easy way to get the spell damage type to resolve on a damage roll since there can be multiple of them